### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by Benmuiruri**
### Blog app - controllers specs
Hello Reviewer :wave: , thank you for taking the time to review my PR. 

# What does this PR do?
:heavy_check_mark: Add users `index` view
:heavy_check_mark: Add posts `show` view
:heavy_check_mark: Install `Bootstrap`
:heavy_check_mark: Add `navbar`
:heavy_check_mark: Add users `show` view
:heavy_check_mark: Add posts `show` view
:heavy_check_mark: Refactor `user_spec` tests
:heavy_check_mark: Refactor `post_spec` tests
 
# Project requirements completed 

- Create a method called current_user in your ApplicationController to make current user data available to all controllers.
- Create forms to perform the following functions:
  1. Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
  2. Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

# How should it be tested
- checkout the branch
`git checkout forms`
-install gems
 `bundle install`
-run tests
`rspec --format documentation`
- start server
 `rails s`

### Thanks for your time and effort to review my code.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,12 @@
...
\ No newline at end of file
+<h2><%= @user.name%>'s Most Recent Posts </h2>
+
+<%= render partial: "shared/user/user_header", locals: { user: @user } %>
+<%= render partial: "shared/user/user_bio", locals: { user: @user } %>
+
+<% @recent_posts.order(comments_counter: :DESC).each do |post| %>
+  <%= render partial: "shared/post/post_card", locals: { post: post } %>
+<% end %>
+
```
Comment (Deepakdanger) [inline app/views/users/show.html.erb:9]: - [ ] In the user's show page, you just have to show the number of comments and likes and not the details of the comments.

### User's show page (Given in wireframe)
Screenshot from 2022-07-13 16-07-11



### Your user's show page

Screenshot from 2022-07-13 16-04-01

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Benmuiruri, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks good :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (Benmuiruri) [reply app/views/users/show.html.erb:9]: Hello @Deepakdanger thanks, I'll make that change

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Benmuiruri,

✔️ Implementation of wireframes

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**